src/components/Board/BoardTour/BoardTour.js

```diff
@@ -23,6 +23,44 @@ const joyRideStyles = {
     textColor: '#333',
     width: 500,
     zIndex: 1000
+  },
+  tooltip: {
+    maxHeight: '85vh',
+    overflowY: 'auto'
+  },
+  buttonNext: {
+    backgroundColor: '#aa00ff'
+  },
+  buttonBack: {
+    color: '#aa00ff'
+  },
+  buttonSkip: {
+    color: '#aa00ff'
+  },
+  // Add custom styles for mobile view
+  tooltipContainer: {
+    textAlign: 'center'
+  },
+  tooltipFooter: {
+    position: 'sticky',
+    bottom: -20,
+    backgroundColor: '#eee',
+    padding: '10px 0',
+    marginTop: '10px',
+    borderTop: '1px solid #ddd',
+    width: '100%',
+    display: 'flex',
+    justifyContent: 'space-between'
+  }
+};
+
+// Adjust tooltip maxHeight for mobile
+const mobileLockedHelpStepStyles = {
+  tooltip: {
+    width: '90vw',
+    maxWidth: 700,
+    maxHeight: '80vh',
+    overflowY: 'auto'
   }
 };
```

Comment:
Take a look at this suggestion. By setting the tooltip as a flex column container, we can better control the positioning of its children. Additionally, applying overflow handling to the tooltipContainer helps resolve the layout issue effectively.

```suggestion
const mobileLockedHelpStepStyles = {
  tooltip: {
    width: '90vw',
    maxWidth: 700,
    maxHeight: '95vh',
    display: 'flex',
    flexDirection: 'column'
  },
  tooltipContainer: {
    overflowY: 'auto'
  },
  tooltipContent: {
    padding: 0
  }
};
```